fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -237,7 +238,8 @@ public Coordinator(ConnectContext context, Analyzer analyzer, Planner planner) {
         this.queryId = context.queryId();
         this.fragments = planner.getFragments();
         this.scanNodes = planner.getScanNodes();
-        this.descTable = analyzer.getDescTbl().toThrift();
+        this.descTable = planner instanceof NereidsPlanner ? planner.getDescTable().toThrift()
+                : analyzer.getDescTbl().toThrift();
```

Comment:
both NereidsPlanner and OriginalPlanner already implement interface getDescTable, we could call getDescTable for both of them